sw/device/tests/power_virus_systemtest.c

```diff
@@ -321,6 +355,39 @@ static void configure_kmac() {
       &kKmacKey, &kmac_customization_string));
 }
 
+static void configure_uart(dif_uart_t *uart) {
+  CHECK_DIF_OK(
+      dif_uart_configure(uart, (dif_uart_config_t){
+                                   .baudrate = kUartBaudrate,
+                                   .clk_freq_hz = kClockFreqPeripheralHz,
+                                   .parity_enable = kDifToggleEnabled,
+                                   .parity = kDifUartParityEven,
+                               }));
+  CHECK_DIF_OK(
+      dif_uart_loopback_set(uart, kDifUartLoopbackSystem, kDifToggleEnabled));
+}
+
+static void configure_i2c(dif_i2c_t *i2c, uint8_t device_addr_0,
+                          uint8_t device_addr_1) {
+  uint32_t peripheral_clock_period_ns =
+      udiv64_slow(1000000000, kClockFreqPeripheralHz, NULL);
+  dif_i2c_config_t config;
+
+  CHECK_DIF_OK(dif_i2c_compute_timing(
+      (dif_i2c_timing_config_t){
+          .lowest_target_device_speed = kDifI2cSpeedFastPlus,
+          .clock_period_nanos = peripheral_clock_period_ns,
+          .sda_rise_nanos = kI2cSdaRiseFallTimeNs,
+          .sda_fall_nanos = kI2cSdaRiseFallTimeNs,
+          .scl_period_nanos = kI2cSclPeriodNs},
+      &config));
+  CHECK_DIF_OK(dif_i2c_configure(i2c, config));
+  dif_i2c_id_t id_0 = {.mask = kI2cDeviceMask, .address = device_addr_0},
+               id_1 = {.mask = kI2cDeviceMask, .address = device_addr_1};
```

Comment:
nit
```suggestion
  dif_i2c_id_t id_0 = {.mask = kI2cDeviceMask, .address = device_addr_0};
  dif_i2c_id_t id_1 = {.mask = kI2cDeviceMask, .address = device_addr_1};
```

Reply:
Technically, our style guide [allows this](https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions) ("_It is allowed (if unusual) to declare multiple variables in the same declaration, but it is disallowed if any of those have pointer or reference decorations._"), but fixed it anyway :)